profiles/base.yaml

```diff
@@ -233,8 +233,11 @@ steal_past_mindlock: false
 stealing_buffs:
 npc_stealing_attempt_count: 0
 dont_steal_list:
-
-
+bin_stolen: false
+bin_location: 9080 #Crossing - 9080 # Muspar'i - 12283
```

Comment:
Is this something that could be in base-town? Or maybe have a mapping of your current stealing town <-> bin room ID?